README.md

```diff
@@ -8,7 +8,8 @@
 2. [Unofficial communities](#unofficial-communities)
 3. [Tools](#tools)
 4. [Addons](#addons)
-    1. [Streams](#streams)
+    1. [Movie & series](#Movie-&-series)
+    2. [Subtitles](#subtitles)
```

Comment:
👍 Makes sense!

Few things:
1. Could "TV shows" be a better word instead of series? It seems to be used more often in existing English media, for example on IMDb.
2. Does the `#Movie-&-series` link work on a rendered Markdown page, did you try it?
3. You also added a `### TV channels` header, maybe that should be included here as well?

Reply:
1. Ahh, great call!
2. Normally spaces can be replaced by hyphens in markdown. It works in my MD editor but it seems like github doesn't work with the `&` sign, so it should become `#Movie--series` (or `#Movie---TV-shows`) in order for it to work.
3. 🤦 added.